oam/install_scripts/columnstore-post-install.in

```diff
@@ -235,11 +243,9 @@ fi
 
 checkForError
 if [ $? -ne 0 ]; then
-    echo "ERROR: Invalid password in .my.cnf, or Columnstore plugin install missing"
-fi
-
-if [ $? -ne 0 ]; then
-    echo "ERROR: Invalid password in .my.cnf, or Columnstore plugin install missing"
+    echo "There was an error installing MariaDB ColumnStore engine plugin. \
+Continue to install the engine though. \ 
+Please resolve the issues and run necessary scripts manually."
```

Comment:
I don't think this is what we mean to say.  I'd offer a suggestion but I'm not completely sure what we're trying to say.  We're telling them to continue installing the engine, to resolve 'issues', and to run 'scripts'.  Can we give them more specific or useful info?  Maybe we suggest a log file to look at to explain what went wrong?  Maybe we tell them what scripts to run?  What can we do here?